### DISCUSSION TITLE
Discussion on adding a `CODEOWNERS` file

### DISCUSSION THREAD

**Discussion by thunder-coding** [termux/termux-packages#7712] [Ideas]
GitHub supports additing a `CODEOWNERS` file https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners. Whenever someone other than the owner edits the file then GitHub automatically asks for review from the owners of the file on PR. This can be very helpful if done properly. 

For example when someone other than package maintainer does some change to packages. It's always good to take some review from the existing maintainer. 

If everyone is ready for this change, I may pull up a PR reflecting this.

**Comment by ghost** [comment]
> If everyone is ready for this change

\+1, ready

**Reply by ghost** [reply]
https://github.com/termux/termux-packages/pull/7716
